### ISSUE TITLE
Labeler Limit : Moderation vs Message

### ISSUE THREAD

**Issue by PSingletary** [bluesky-social/social-app#4515]
**Describe the bug**
On iOS, the Settings > Moderation Advanced shows the Labelers used/followed, the limit is 9 (BlueSky Moderation Service + 8 more Labelers).

**To Reproduce**

1. subscribe to 8 Labelers 
2. Try to subscribe to a 9th labeler
3. The button is greyed out, and tapping on it informs that you can only subscribe to 10 Labelers

**Expected behavior**

Subscribe to 9th Labeler

**Screenshots**

Will update with edit

**Details**

Build version: 1.86.0.290; Bundle info: 6736384 (prod); Bundle date: 24061123; Platform: ios

**Additional context**

Will test with Microsoft Edge and update

**Comment by pfrazee** [comment]
Yeah gonna fix this shortly, we're bumping the limit and I'll make sure the message is correct

**Comment by surfdude29** [comment]
This has now been fixed on [main.bsky.dev](https://main.bsky.dev/) by #4565. I'm able to subscribe to 20 labellers in addition to Bluesky Moderation Service.

**Comment by pixiekat** [comment]
Any chance to bump it higher? With cosmetic and moderator labellers, I'm hitting the limit of 20 already. Or maybe allow more to be added but add the option to disable/enable labeller so you can be subscribed to more than 20 but only have 20 max *enabled* at once? 

Cosmetic labellers are fun and people are really creative with them but between them and moderation labels, it's sad that I have to choose between the two.

**Comment by cooljeanius** [comment]
> validated [#4515 (comment)](https://github.com/bluesky-social/social-app/issues/4515#issuecomment-2180548375) closing

...this seems to still be open, though?
